cmd/minishift/cmd/root.go

```diff
@@ -123,6 +126,10 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
+		if hasEnabledExperimental {
+			fmt.Println("!!! Experimental features are enabled !!!")
```

Comment:
It prints that you enabled it... especially for the situation where you have set it from your bash profile. At the moment we do not change behaviour... but maybe we can instead log it.